### PR TITLE
feat(route): add route for graduacted school of CJLU (China Jiliang University)

### PR DESCRIPTION
**Pull request by chrisis58**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/cjlu/yjsy/yjstz
/cjlu/yjsy/jstz
```

## New RSS Route Checklist / 新 RSS 路由检查表

- [x] New Route / 新的路由
  - [x] Follows [[Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard)](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [[路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [[Date and time](https://docs.rsshub.app/joinus/advanced/pub-date)](https://docs.rsshub.app/joinus/advanced/pub-date) / [[日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

为中国计量大学研究生院添加路由。

其中 `yjsy` 为中文`研究生院`的拼音首字母缩写。

同理 `yjstz` 和 `jstz` 为`研究生通知`和`教师通知`的缩写。

### CODE REVIEW

File: lib/routes/cjlu/yjsy/index.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['yjsy.cjlu.edu.cn/index/:cate/:id.htm'],
```
Comment (TonyRL) [inline lib/routes/cjlu/yjsy/index.ts:33]: `/:id` will match `.htm` in the url path as well

File: lib/routes/cjlu/yjsy/index.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            target: '/yjsy/:cate',
+        },
+        {
+            source: ['yjsy.cjlu.edu.cn/index/:cate.htm'],
```
Comment (TonyRL) [inline lib/routes/cjlu/yjsy/index.ts:37]: `:cate` will match `.htm` in the url path as well

File: lib/routes/cjlu/yjsy/index.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            target: '/yjsy/:cate',
+        },
+    ],
+    name: '中量大研究生院',
```
Comment (TonyRL) [inline lib/routes/cjlu/yjsy/index.ts:41]:
```suggestion
    name: '研究生院',
```

File: lib/routes/cjlu/yjsy/index.ts (hunk 1 above)
Comment (chrisis58) [reply lib/routes/cjlu/yjsy/index.ts:33]: Sorry for my absence during the day. I've tried patterns like `yjsy.cjlu.edu.cn/index/:cate{[a-z]+}/:id{[0-9]+}.htm` and `yjsy.cjlu.edu.cn/index/:cate{[a-z]+}.htm`, but they don't seem to work as expected.

I also referred to an existing route (lib/routes/sjtu/seiee), and here’s the relevant code:
```typescript
    radar: [
        {
            source: ['www.seiee.sjtu.edu.cn/:path.html'],
            target: '/seiee/:path',
        },
    ],
```
Could you point me in the right direction? I would greatly appreciate any suggestions!

Comment (chrisis58) [reply lib/routes/cjlu/yjsy/index.ts:33]: Additionally, there're some exapmles:
```
https://yjsy.cjlu.edu.cn/index/yjstz.htm // notifications for students page 1 of 93 (fitst page). 
https://yjsy.cjlu.edu.cn/index/yjstz/92.htm // notifications  for students page 2 of 93.
https://yjsy.cjlu.edu.cn/index/yjstz/1.htm // notifications for students page 93 of 93 (last page).

https://yjsy.cjlu.edu.cn/index/jstz.htm // notifications for teachers page 1 of 17 (first page).
https://yjsy.cjlu.edu.cn/index/jstz/16.htm // notifications for teacher page 2 of 17.
https://yjsy.cjlu.edu.cn/index/jstz/1.htm // notifications for teacher page 17 of 17 (last page).
```

Comment (TonyRL) [reply lib/routes/cjlu/yjsy/index.ts]: Hard-code `yjsy.cjlu.edu.cn/index/yjstz.htm` which targets `/yjsy/yjstz` and `yjsy.cjlu.edu.cn/index/jstz.htm` with `/yjsy/jstz`

File: lib/routes/cjlu/yjsy/index.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+const titleMap = new Map([
+    ['yjstz', '中量大研究生院 —— 研究生通知'],
+    ['jstz', '中量大研究生院 —— 教师通知'],
+]);
+
+export const route: Route = {
+    path: '/yjsy/:cate',
+    categories: ['university'],
+    example: '/cjlu/yjstz',
```
Comment (TonyRL) [inline lib/routes/cjlu/yjsy/index.ts:18]: This example won't work since `cate` is defined as compulsory parameter

Comment (chrisis58) [reply lib/routes/cjlu/yjsy/index.ts]: Thanks for your reviewing! I have corrented the example of this route.

File: lib/routes/cjlu/yjsy/index.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            title: '研究生通知',
+            source: ['yjsy.cjlu.edu.cn/index/yjstz/:suffix', 'yjsy.cjlu.edu.cn/index/yjstz.htm'],
+            target: '/yjsy/yjstz',
+        },
+        {
+            title: '教师通知',
+            source: ['yjsy.cjlu.edu.cn/index/jstz/:suffix', 'yjsy.cjlu.edu.cn/index/jstz.htm'],
+            target: '/yjsy/jstz',
+        },
+    ],
```
Comment (chrisis58) [inline lib/routes/cjlu/yjsy/index.ts:42]: Since there're only two choices of categories, I list them all in the radar rules.

**Pull request merged**